server/tournaments/index.ts

```diff
@@ -1263,6 +1287,30 @@ function createTournament(
 }
 
 const commands: Chat.ChatCommands = {
+	pasttours: 'recenttours',
+	recenttours(target, room, user) {
+		this.runBroadcast();
+		room = this.requireRoom();
+		if (!room.settings.tournaments?.recentToursLength) {
+			throw new Chat.ErrorMessage(`Recent tournaments aren't documented in this room.`);
+		}
+		if (!room.settings.tournaments?.recentTours?.length) {
+			throw new Chat.ErrorMessage(`There haven't been any documented tournaments in this room recently.`);
+		}
+		// Shorten array if the recentToursLength gets adjusted
+		const array = room.settings.tournaments.recentTours.filter((x, i) => (
+			i <= room!.settings.tournaments!.recentToursLength! - 1
+		));
```

Comment:
If `recentTourLength` has changed, why not either remove the elements from `recentTours` (instead of making a new array), or just trim `recentTours` when you change `recentTourLength`?

Reply:
I figured people might want to preserve it while coding but looking back I don't know why I thought that since it seems pretty minute, so I'll fix it to do the latter